rotkehlchen/chain/ethereum/modules/eth2/utils.py

```diff
@@ -137,6 +137,16 @@ def scrape_validator_daily_stats(
                 column_pos += 1
             elif column_pos == 2:
                 pnl = _parse_fval(column.string, 'income')
+                # if the validator makes profit in the first day beaconchain returns a
+                # profit of deposit + validation reward. We need to subtract the deposit value
+                # to obtain the actual pnl.
+                # Example: https://beaconcha.in/validator/999/stats
+                if pnl > ONE:
+                    # Since the profit in one day is < 1 ETH get the integer part
```

Comment:
Does this happen only for stakers that are active at genesis? If a validator starts later will the 32 ETH deposit also show like this? If yes then I can see edge cases where this breaks